src/ott/solvers/outputs.py

```diff
@@ -0,0 +1,66 @@
+from typing import Optional, Tuple
+
+import jax.numpy as jnp
+from flax import struct
+
+from ott.geometry import geometry
+from ott.problems.linear import linear_problem
+
+
+@struct.dataclass
+class _BaseTransportOutput:
+  """Base class for Transport output.
+
+  Args:
+    ot_prob: OT problem.
+    errors: Holds sequence of vectors of errors of the Sinkhorn algorithm
```

Comment:
I think this a nice opportunity to fix `errors/costs`:

- `Sinkhorn` currently stores only `errors` which determine the convergence
- `LRSinkhorn` stores also stores the `reg_ot_cost` in `costs`

I think we can change it so that in `_BaseTransportOutput` always has `errors` and optionally have `costs`; pinging @marcocuturi for opinion.